R/deployApp.R

```diff
@@ -418,15 +415,15 @@ deployApp <- function(appDir = getwd(),
         application
       },
       rsconnect_http_404 = function(err) {
-        application <- applicationDeleted(client, target, recordPath, appMetadata)
+        application <- applicationDeleted(client, deployment, recordPath, appMetadata)
         taskComplete(quiet, "Created application with id {.val {application$id}}")
         application
       }
     )
   }
   saveDeployment(
     recordPath,
-    target = target,
+    target = deployment,
```

Comment:
note: This is the remaining use of `target` that is a deployment record. Consider renaming the argument, though that means renaming the in-function variable, as well.

Reply:
note: The 0.8.29 implementation wrote deployment records only when the target (local) account was the content owner, using username matching. This PR does not revert to that behavior; all deploys write to disk, though we try to maintain the account+server whenever possible.

Reply:
The `target` has been renamed.